src/core/qgsmaptopixelgeometrysimplifier.cpp

```diff
@@ -18,9 +18,10 @@
 #include "qgsmaptopixelgeometrysimplifier.h"
 #include "qgsapplication.h"
 
-QgsMapToPixelSimplifier::QgsMapToPixelSimplifier( int simplifyFlags, double tolerance )
+QgsMapToPixelSimplifier::QgsMapToPixelSimplifier( int simplifyFlags, double tolerance, int simplifyAlgorithm )
```

Comment:
Sure, but this pull is only a test to resolve some existing issues. Do you agree postpone it up to verify better results than current status ?